src/components/Content/TaskHierarchy/Task/Task.css

```diff
@@ -77,6 +77,14 @@
   padding: 0 8px;
 }
 
+.icon-ctn svg {
+  color: #23527c;
+}
+
+.icon-ctn:hover svg {
+  color: #288bd6;
+}
```

Comment:
@MartyTheeMartian Please swap these 2 colors, so that it's nice and bright and easier to see (higher contrast) by default, and dims when we hover